transformer_engine/jax/csrc/modules.cpp

```diff
@@ -37,6 +37,19 @@ std::vector<size_t> MakeShapeVector(NVTEShape shape) {
     return std::vector<size_t>(shape.data, shape.data + shape.ndim);
 }
 
+size_t get_activation_len(NVTE_Activation_Enum act_enum) {
+  switch (act_enum) {
+    case NVTE_Activation_Enum::GELU: return 1;
+    case NVTE_Activation_Enum::GEGLU: return 2;
+    case NVTE_Activation_Enum::SILU: return 1;
+    case NVTE_Activation_Enum::SWIGLU: return 2;
+    default:
+      NVTE_ERROR("Unsupported ActivationEnum");
+      break;
+    return -1;
+  }
+}
+
```

Comment:
@zlsh80826 Do you have a better way to extract `act_len` here?

Reply:
This looks good to me!